ansible/roles/docker/handlers/main.yml

```diff
@@ -1,3 +1,50 @@
 ---
 - name: restart docker
-  service: name=docker state=restarted
+  command: /bin/true
+  notify:
+    - check docker existence
+    - stop docker
+    - delete docker0
+    - start docker
+
+- name: start docker
+  command: /bin/true
+  notify:
+    - check docker existence
+    - docker graph driver
+    - docker fix start on btrfs
+    - docker start service
+
+- name: check docker existence
```

Comment:
If we are running the docker role, doesn't that mean docker must be installed (or we would have installed it?)